src/sinks/prometheus/exporter.rs

```diff
@@ -58,7 +55,7 @@ enum BuildError {
 
 /// Configuration for the `prometheus_exporter` sink.
 #[serde_as]
-#[configurable_component(sink)]
+#[configurable_component(sink("prometheus_exporter"))]
```

Comment:
Remind me, we're fully deprecating this - or are we providing the old name elsewhere that I'm missing.